src/category_theory/opposites.lean

```diff
@@ -6,6 +6,20 @@ Authors: Stephen Morgan, Scott Morrison
 import category_theory.types
 import category_theory.equivalence
 
+/-!
+# Opposite categories
+
+We provide a category instance on `Cᵒᵖ`.
```

Comment:
Can you quickly explain what's the opposite category?

Reply:
```suggestion
We provide a category instance on `Cᵒᵖ`.
The morphisms `X ⟶ Y` are defined to be the morphisms `unop Y ⟶ unop X` in `C`.

```

Reply:
Perfect!